base_layer/wallet/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![recursion_limit = "512"]
+#![recursion_limit = "1024"]
```

Comment:
😲

Reply:
Oh, I should change this back. It came from a mistake I made in one of my `select!`s that I fixed later